[REG-1293] Update Agent Builder documentation for Unity SDK (#39)
Regression-Games/RegressionDocs: docs/common/agent-builder/_adding-nodes.mdx

> Whenever you create a fresh Agent Builder project for your AI, we'll start you out with a root node and a sequence node to form the basis of your tree.

Reviewer: I think this can be a bit more concise

```suggestion
New Agent Builder projects start with a root node and a sequence node to form the basis of your tree.
```

Author: Updated
